### ISSUE TITLE
Mine causing crash bug...

### ISSUE THREAD

**Issue by YassevZriona** [Geforce132/SecurityCraft#297]
Placing two mines or more together -> Using a mine bounder to bound the two mines -> Using the bounder to detonate one mine while causing the one adjacent to it to be broken while still being bounded -> Using the mine bounder, Detonate the mine which was broken by the detonation of another mine -> Game crash and perhaps even server crash

Client Version: 1.12.2 Forge latest version 
Mod Version: 1.12.2 Latest Release

Also as an extra question - is the bug fixed in which if a player opened a camera using the monitor then he would be teleported as a spectator or something there which could be abused while being in a fight...

**Comment by bl4ckscor3** [comment]
1. Please don't say "latest version". We don't know if what you think is the latest version is actually the latest version. Always use explicit version numbers
2. When reporting a crash, always include a crash report. Reproduction steps are a good start, but a crash report contains additional information that could become useful. In this case, I cannot reproduce the bug with the steps you provided, so the crash report will be very useful.
3. It's called "Mine Remote Access Tool", not "Mine Bounder"
4. The camera thing is not a bug, but intended behavior. However in favor of being able to view cameras over long distances, we've changed this for the next update.

**Comment by YassevZriona** [comment]
Okay I am such a noob in making bug reports 😂
I seem to have deleted the crashreport before and on phone rn so will provide it later...
Please can you add me on discord? Yassev#6631

**Comment by bl4ckscor3** [comment]
No problem. Once you're back on your PC, just recreate the crash send the crash report of that one here as a reply. I don't take friend requests on Discord.

**Comment by YassevZriona** [comment]
Hmm so I tried doing it again but it doesn't seem to happen now, I am sure I am using the same security craft and forge version. It seems like it was some different bug perhaps caused by a lag in which from what I remember I did see 1 mine being detonated while others who were in the explosion radius didn't detonate, all explosion damage was just at the east side and not the west side, after that when I detonated another bomb it crashed.
Well If I do come by this again I will make sure to find the issue myself properly and retest at least! Hope you have a good day even with my bothers.
